controllers/suite_util_test.go

```diff
@@ -592,6 +604,32 @@ func createCAConfigMap(configMapKey types.NamespacedName) {
 	}
 }
 
+func createCustomRenovateConfigMap(configMapKey types.NamespacedName, configMapData map[string]string) {
+	customConfigMap := corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{Name: configMapKey.Name, Namespace: configMapKey.Namespace},
+		Data:       configMapData,
+	}
+	if err := k8sClient.Create(ctx, &customConfigMap); err != nil && !k8sErrors.IsAlreadyExists(err) {
+		Fail(err.Error())
+	}
+}
+
+func deleteConfigMap(configMapKey types.NamespacedName) {
+	configMap := corev1.ConfigMap{}
+	if err := k8sClient.Get(ctx, configMapKey, &configMap); err != nil {
```

Comment:
Do we really need to check the if Config Map exists?

Reply:
yes, to prevent any conflicts

Reply:
Which conflicts do you mean? You already check the error on delete call.

Reply:
conflicts with other tests, I need to be sure that configMap is really gone, because otherwise old object could be used with other test

Reply:
we need to first get the object in order to delete it, as delete requires object and not just a name